code/src/index.css

```diff
@@ -1,13 +1,160 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans&family=Quicksand&display=swap');
+
 body {
+  color: #ffffff;
+  font-family:'Quicksand', sans-serif; ;
+  display: flex;
+  justify-content: center;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  background-image: url(./Images/travel.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100vh;
+  width: 100vw;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+ h1 {
+  font-size: 35px;
+  font-weight: bold;
+  text-align: center;
+ }
+
+ h2 {
+  font-size: 30px; 
+  text-align: center;
+  width: 350px;
+ }
+
+ p {
+  font-size: 20px;
+  width: 450px;
+ }
+
+/* FORM */
+
+form {
+  margin-top: 70px;
+  height: 400px;
+  width: 550px;
+  padding: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  background: rgb(44 43 43 / 80%);
+  box-shadow: 20px 20px 30px gray;
+  border-radius: 3%;
 }
+
+/* BUTTON */
+
+.button-container  {
+  display: flex;
+  justify-content: center;
+}
+
+button {
+  font-family: 'Nunito Sans', sans-serif;
+  font-size: 16px;
+  text-align: center;
+  padding: 5px;
+  margin: 10px;
+  width: 100px;
+  border-radius: 5px;
+  color: #ffffff;
+  background: transparent;
+}
+
+button:hover {
+  cursor: pointer;
+}
+/* INPUT */
+
+.input {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+input {
```

Comment:
Perhaps next time you could style the input:invalid a bit? At the moment, since you have a very dark background, it was a little bit hard to see when the input was invalid with the default styling. Just a tip for the future 🥰